scipy/stats/tests/test_stats.py

```diff
@@ -501,6 +505,11 @@ class TestCorrSpearmanr(TestCase):
         other variables.  The same should go for SPEARMAN corelations, if
         your program has them.
     """
+    def test_scalar(self):
+        y = stats.spearmanr(4.)
+        r = y[0]
+        assert(np.isnan(r))
```

Comment:
This should be ``assert_``. Reason: ``assert`` statements get stripped when byte-compiling with ``-O``, so we don't use them for tests.